### PR TITLE
Resolve Broken Link Issues

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Fix broken links in the Repo.

### CODE REVIEW

**Review by jsquire — APPROVED** [review]

**Review by weshaggard — APPROVED** [review]

**Comment by weshaggard** [comment]
@chidozieononiwu what is causing you to submit this PR? Is there an issue or some report you are working off?

**Comment by chidozieononiwu** [comment]
> @chidozieononiwu what is causing you to submit this PR? Is there an issue or some report you are working off?

No particular related issue. Just trying to keep aggregate reports pipeline green. Which I do periodically.

**Review by weshaggard — APPROVED** [review]

**Comment by weshaggard** [comment]
@jsquire @pallavit how do you plan to have folks from your teams monitor the aggregate-reports pipeline to make sure that the overall health of the repo is clean of things like this?

File: sdk/search/Microsoft.Azure.Search/CHANGELOG.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ None
 
 ### Breaking Changes
 
-For more details, visit the [SDK migration guide](https://docs.microsoft.com/azure/search/search-dotnet-sdk-migration-version-9).
+For more details, visit the SDK migration guide.
```
Comment (benbp) [inline sdk/search/Microsoft.Azure.Search/CHANGELOG.md:102]: Are these links just gone? I'd think we would still want to link to _something_ if we can.

Comment (chidozieononiwu) [reply sdk/search/Microsoft.Azure.Search/CHANGELOG.md:102]: the docs were archived but for some reason they don't show up in previous versions. They may eventually be fixed but there is not estimate as to when.

Comment (benbp) [reply sdk/search/Microsoft.Azure.Search/CHANGELOG.md:102]: The version is now version 11 and the link is here: https://docs.microsoft.com/en-us/azure/search/search-dotnet-sdk-migration-version-11. There are also hard links next to that page in the TOC for upgrading the rest api and mgmt sdk, but sadly the data plane sdk does not have a consistent link. @weshaggard @danieljurek does this seem like an oversight or something we can fix? I'm not the biggest fan of having to choose between links that will expire or referencing docs without a link and making the user find them.

**Comment by weshaggard** [comment]
Yes as @chidozieononiwu says the aggregate-reports pipeline is a nightly pipeline that does some reports and compliance stuff across the entire repo. We try and do as much as we can in an individual PR but there are somethings that cannot be ran or we need to run across the entire repo so we scope it to that pipeline. We have an entry for it in the codeowners file, https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS#L703 to identify owners for the notification emails, it looks like you added yourself there with the recent update. Some teams appoint an owner or have a rotating owner to watch the results of this pipeline to make sure the repo stays in a good state.

**Pull request merged**